requirements.txt

```diff
@@ -1,3 +1,3 @@
 cython
 virtualenv
-git+https://github.com/endlessm/python-for-android@v2022.09.04-endless1#egg=python-for-android
+git+https://github.com/endlessm/python-for-android@8a506eff5261635d987bc9b578a021c7d84d22e5#egg=python-for-android
```

Comment:
This is a pointer to https://github.com/endlessm/python-for-android/pull/11. If it works we'd want to merge that PR, make a `v2022.09.04-endless2` tag, and update this accordingly.